.travis.yml

```diff
@@ -19,6 +21,13 @@ env:
   - GCLOUD=$HOME/gcloud/google-cloud-sdk/bin 
   - CLOUDSDK_PYTHON_SITEPACKAGES=1 
   - GOOGLE_APPLICATION_CREDENTIALS=$TRAVIS_BUILD_DIR/servicekey.json
+  #GenomicsDB Stuff
+  - GATK_GENOMICSDB_BIN=src/test/resources/large/tiledb_bin
```

Comment:
As @lbergelson said, hopefully when we have a write API it will eliminate the need to check in these extra binaries.